PublicGitArchive/doc/dataset_analysis.md

```diff
@@ -0,0 +1,70 @@
+# Description of the current dataset
+
+## 1. GHTorrent: selecting the list of URLs
+
+From GHTorrent’s MySQL dump, dated from 01/01/2018.
+By querying repos with at least 50 stargazers
+
+Output: list of 190,051 URLs with duplicates  
```

Comment:
I changed the code to output the unique URLs directly, so it is impossible to check. I propose to remove this number.